questions.md

```diff
@@ -0,0 +1,124 @@
+<p align="center">
+<img src="logo.png">
+</p>
+
+
+babylon health
+==================================
+
+As described in the iOS recruitment process [main page](TODO), the face-to-face technical interviews involve us asking candidates a number of technical questions which we have decided to open-source.
```

Comment:
face-to-face or skype

Reply:
Well, Skype is still face-to-face, just not in person. 😛